XRMSolutionDependencyChecker/PluginControl.cs

```diff
@@ -282,7 +301,8 @@ public string ShowMissingComponents(byte[] ExportedSolution)
                     tComp.Columns.Remove("Required Component Parent Schema Name");
                     gridWidth -= 400;
```

Comment:
I am a bit confused why it appears that we are hardcoding width?

Reply:
Forgot to remove the hardcoding of subtracting the width depending on checkboxes (this was there in the original design)